src/coreclr/System.Private.CoreLib/Tools/GenUnicodeProp/CharUnicodeInfoData.cs

```diff
@@ -0,0 +1,2424 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Don't check in this file. Only need to update the file in the normal source directory.

Reply:
Oh right. this was a mistake. I'll revert it :-)